docs/content/v0.20/packaging.md

```diff
@@ -1,5 +1,26 @@
 # Packaging and Delivery
 
+## Outline of GitHub repositories
+
+The following is a graph of the GitHub repositories and the dependencies among them. The repo at the tail of an arrow depends on the repo at the head of the arrow.
+
+```mermaid
+flowchart LR
+    kubestellar --> kubeflex
+    kubeflex    --> kubestellar
+    kubestellar --> ocm-status-addon
+    kubestellar --> ocm-transport-plugin
+    ocm-transport-plugin --> kubestellar
```

Comment:
why `kubestellar` repo depends on `ocm-transport-plugin` repo?

Reply:
as far as I know, `kubestellar` repo do not add reference to `ocm-transport-plugin` repo, and it shouldn't do so.
`ocm-transport-plugin` repo references `kubestellar` repo (in go.mod) cause it uses the generic transport controller (calls the cmd.Run) to initialize ocm based transport controller.

Reply:
Updated.

Reply:
I was also doing a quick search now to find where `kubestellar` references `ocm-status-addon` and I didn't find such reference. 
I think only one direction exists (`ocm-status-addon` references `kubestellar`). this direction makes sense to me.

Reply:
updated again.